lib/td/client.rb

```diff
@@ -111,6 +111,16 @@ def update_schema(db_name, table_name, schema)
     @api.update_schema(db_name, table_name, schema.to_json)
   end
 
+  # @param [String] db
+  # @param [String] table
+  # @option opts [Fixnum] :expire_days days to expire table
+  # @option opts [Boolean] :include_v (true) include v column on Hive
+  # @option opts [Boolean] :detect_schema (true) detect schema on import
+  # @return [true]
+  def update_table(db_name, table_name, params)
```

Comment:
Doc comment called opts but the arg name is params. Maybe the same name is better.